include/gsl/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+cc_library(
```

Comment:
Why is this file in the `include` directory?

Reply:
It's an convention from bazel that regard each folder as a package. The build file is for the contents in this package.